src/App.js

```diff
@@ -1,43 +1,32 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Header } from "./components/header";
+import {
+    HomePage,
+    Login,
+    SignUp,
+    Product,
+    WishList,
+    Cart,
+} from "./pages/Pages";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
+    return (
+        <div className="App">
+            <BrowserRouter>
+                <Header />
+                <Routes>
+                    <Route path="/" element={<HomePage />} />
+                    <Route path="homePage" element={<HomePage />} />
+                    <Route path="login" element={<Login />} />
+                    <Route path="signup" element={<SignUp />} />
+                    <Route path="product" element={<Product />} />
+                    <Route path="wishlist" element={<WishList />} />
+                    <Route path="cart" element={<Cart />} />
```

Comment:
you can add one more route to page not found 404 error

Reply:
I will add it in the upcoming PR.